src/components/ssi/AuthenticationModal/agent.ts

```diff
@@ -0,0 +1,20 @@
+import { createAgent } from '@veramo/core'
+import {
+  IQRCodeGenerator,
+  QrCodeProvider
+} from '@sphereon/ssi-sdk.qr-code-generator'
+import {
+  ISIOPv2OID4VPRPRestClient,
+  SIOPv2OID4VPRPRestClient
+} from '@sphereon/ssi-sdk.siopv2-oid4vp-rp-rest-client'
+
+const agent = createAgent<IQRCodeGenerator & ISIOPv2OID4VPRPRestClient>({
+  plugins: [
+    new QrCodeProvider(),
+    new SIOPv2OID4VPRPRestClient({
+      baseUrl: process.env.NEXT_PUBLIC_SSI_BACKEND_BASE_URL,
```

Comment:
Please doublecheck whether the REST client checks for the baseUrl. Otherwise add a check here

Reply:
It's checked in the REST client